test/csuite/schema_abort/main.c

```diff
@@ -1041,15 +1041,9 @@ main(int argc, char *argv[])
      */
     if (chdir(home) != 0)
         testutil_die(errno, "parent chdir: %s", home);
-    /*
```

Comment:
The comment here is confusing. It is copying all the data not just the log. I noticed in random_abort we are only copying the log files. I changed it also to copy all the data as the other tests seems to be fine.